### PR TITLE
update zh_cn.lang

### PR DESCRIPTION
**Pull request by Demonese**
related pull request: https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/2540

I noticed that the language files for minimap have been separated from betterpvp, this is a good news.

_GraphicKuanlan is also my account._

### CODE REVIEW

**Review by ChuijkYahus — APPROVED** [review]

**Comment by thexaero** [comment]
Thanks, will take a look soon. Is this based on SolidBlock-cn's translation? Their translation seemed really good, was just hoping for a few fixes. If not, I think I'll just merge theirs and remove the unaddressed lines, so other people can translate them.

**Comment by Demonese** [comment]
> Thanks, will take a look soon. Is this based on SolidBlock-cn's translation? Their translation seemed really good, was just hoping for a few fixes. If not, I think I'll just merge theirs and remove the unaddressed lines, so other people can translate them.

This translation is not based on SolidBlock-cn's, because, unfortunately I didn't know that such a translation repository existed before.

I opened the game and translated it while testing, and the accuracy of the translation can be guaranteed. It's a translation done by myself.

The situation of the Minimap is the same.

**Comment by Demonese** [comment]
There is a problem, the diff is still too many. I'll try to sort out the changed parts.

**Comment by Demonese** [comment]
Now you can review the diff :)

**Comment by thexaero** [comment]
Alright, now that SolidBlock-cn's translation has been merged, I can review your additions. 
I'm very confident in the accuracy of SolidBlock-cn's translation as I have already fully reviewed it and was very impressed by it. Please only translate lines that they haven't done or explain each line that you're replacing.
Here is your translation with the correct line order: https://www.dropbox.com/s/6ky10zmz3c44hn4/zh_cn.lang?dl=0

**Comment by Demonese** [comment]
> Here is your translation with the correct line order: https://www.dropbox.com/s/6ky10zmz3c44hn4/zh_cn.lang?dl=0

I have updated my commit.

**Comment by thexaero** [comment]
Awesome, thanks! Like I said, please only add new lines or explain the changes you make to SolidBlock-cn's.

**Comment by Demonese** [comment]
> I'm very confident in the accuracy of SolidBlock-cn's translation as I have already fully reviewed it and was very impressed by it. Please only translate lines that they haven't done or explain each line that you're replacing.

## 1. Standardization of translation

The translations of these entries have been changed to be consistent with the Minecraft Wiki.

* `gui.xaero_box_notifications`
* `gui.xaero_box_quick_use`
* `gui.xaero_hunger_setting`
* `gui.xaero_hunger_low`
* `gui.xaero_hp_low`
* `gui.xaero_air_setting`
* `gui.xaero_air_low`
* `gui.xaero_entity_info_distance`
* `gui.xaero_entity_info_max_hearts`
* `gui.xaero_entity_info_numbers`
* `gui.xaero_entity_info_armour_numbers`
* `gui.xaero_entity_info_armour`

## 2. Consistent with original English

These translated entries are changed to be consistent with the original English. In addition, translation errors were corrected

* `gui.xaero_box_sprint`
* `gui.xaero_box_sneak`
* `gui.xaero_box_notifications`
* `gui.xaero_box_xp`
* `gui.xaero_box_quick_use`
* `gui.xaero_box_minimap`
* `gui.xaero_being_shot_setting`
* `gui.xaero_restores`
* `gui.xaero_entity_info_stay`
* `gui.xaero_show_full_amount`
* `gui.xaero_entity_info_potion_effects`
* `gui.xaero_item_tooltip`
* `gui.xaero_entity_tooltip_settings`
* `gui.xaero_potion_effects_blink`
* `gui.xaero_no_durability`

## 3. Consistent with the official translation style

* `gui.xaero_quick_use_1`
* `gui.xaero_quick_use_2`
* `gui.xaero_quick_use_3`
* `gui.xaero_quick_use_4`
* `gui.xaero_quick_use_5`
* `gui.xaero_quick_use_6`
* `gui.xaero_quick_use_7`
* `gui.xaero_quick_use_8`
* `gui.xaero_quick_use_9`

File: zh_cn.lang [hunk 1]
```diff
@@ -5,27 +5,27 @@ gui.xaero_miscellaneous=杂项
...
-gui.xaero_box_armour_status=一个显示当前装备和主手物品信息的界面。
-gui.xaero_box_sprint=使用一个单独的按键（可配置的）来疾跑。
-gui.xaero_box_sneak=允许你切换潜行状态，并且在潜行状态中不需要按住任何按键。
-gui.xaero_box_notifications=为一些信息启用屏幕通知，如低生命值、低饱食度、低氧气量、爆炸危险以及被射箭的通知。
-gui.xaero_box_xp=获得或者失去经验值时将会有提示。
-gui.xaero_box_quick_use=一个类似于动作条的热键系统，可以让你在不将主手物品换掉的情况下使用指定的物品。按住热键来持续使用物品，就像按住右键时一样。比如：放置火把、TNT、扔药水、吃东西、喝东西等等。
-gui.xaero_box_minimap=一个显示你周围环境与实体的小地图。
+gui.xaero_box_potion_effects=显示当前激活的药水效果的界面。
+gui.xaero_box_armour_status=显示你当前穿的盔甲和持有物品信息的界面。
+gui.xaero_box_sprint=通过按下一个绑定的键（可配置）来疾跑。
```
Comment (thexaero) [inline zh_cn.lang]: The old translation seems more correct to me when translated back. The main point of this feature is that you only use 1 key to sprint and don't have to also hold the W key to move forward.

File: zh_cn.lang [hunk 2]
```diff
@@ -35,43 +35,43 @@ gui.xaero_numeral=数值
...
+gui.xaero_box_entity_info=显示你所指向的生物实体的信息，包括玩家。
 gui.xaero_entity_info_settings=实体信息设置
-gui.xaero_entity_info_stay=继续显示实体信息
-gui.xaero_entity_info_distance=可达距离
-gui.xaero_entity_info_max_hearts=Max Heart Number
-gui.xaero_entity_info_numbers=Health in Numbers
-gui.xaero_show_full_amount=Full Held Item Quantity
+gui.xaero_entity_info_stay=继续显示
+gui.xaero_entity_info_distance=触及距离
+gui.xaero_entity_info_max_hearts=最大生命值
```
Comment (thexaero) [inline zh_cn.lang]: This is supposed to say "hearts" specifically, not health. If an entity has 100 hearts of health and this is set to 10, it'll display 10 hearts. At 50/100 real hearts, it'll display 5/10 hearts.

File: zh_cn.lang [hunk 3]
```diff
@@ -35,43 +35,43 @@ gui.xaero_numeral=数值
...
+gui.xaero_entity_info_numbers=生命值
```
Comment (thexaero) [inline zh_cn.lang]: Please make sure this works for a toggle between showing hearts and showing numbers.

File: zh_cn.lang [hunk 4]
```diff
@@ -35,43 +35,43 @@ gui.xaero_numeral=数值
...
+gui.xaero_show_full_amount=手持物品的总数
```
Comment (thexaero) [inline zh_cn.lang]: Doesn't seem correct. Please make sure this means "the sum of all item stacks in your inventory for the same item type as you're holding"

File: zh_cn.lang [hunk 5]
```diff
@@ -35,43 +35,43 @@ gui.xaero_numeral=数值
...
 gui.xaero_show_entity_model=显示实体模型
-gui.xaero_entity_info_armour_numbers=盔甲数值
-gui.xaero_entity_info_armour=显示盔甲信息
-gui.xaero_entity_info_potion_effects=药水效果（仅单人）
+gui.xaero_entity_info_armour_numbers=护甲值
```
Comment (thexaero) [inline zh_cn.lang]: Please make sure this works for a toggle between showing armor icons and showing numbers. I don't think it was correct originally either. Not sure how I missed that.

**Review by thexaero — COMMENTED** [review]
Great, thank you, I think you have actually improved it. I left some comments though.

File: zh_cn.lang (hunk 5 above)
Comment (Demonese) [reply zh_cn.lang]: Yes, so i corrected it.

Update: already fixed

File: zh_cn.lang (hunk 4 above)
Comment (Demonese) [reply zh_cn.lang]: > the sum of all item stacks in your inventory for the same item type as you're holding

I do realize that. If the Chinese translation is still not enough to fully explain, I will update it.

File: zh_cn.lang (hunk 3 above)
Comment (Demonese) [reply zh_cn.lang]: already fixed

File: zh_cn.lang (hunk 2 above)
Comment (Demonese) [reply zh_cn.lang]: OK, I will bring more information in the translation

File: zh_cn.lang (hunk 4 above)
Comment (Demonese) [reply zh_cn.lang]: After thinking about it, I decided to change the "总数 (total number)" to "总携带数量 (total carry amount)".

My other Chinese translator friends also support this modification :)

File: zh_cn.lang (hunk 1 above)
Comment (Demonese) [reply zh_cn.lang]: I edited the translation to emphasize that sprinting only requires one key press :)

**Comment by thexaero** [comment]
Awesome, looks good to me! Really appreciate your contribution. Let me know when you want me to merge.

**Comment by Demonese** [comment]
> Awesome, looks good to me! Really appreciate your contribution. Let me know when you want me to merge.

It can be merged now, its quality is high enough.
Thank you for developing the mod :)

**Comment by thexaero** [comment]
Alright, going to do it then. Thanks for the support :)

**Pull request merged**